casepro/settings_common.py

```diff
@@ -462,7 +465,10 @@
 LOGIN_REDIRECT_URL = "/"
 LOGOUT_REDIRECT_URL = "/"
 
-AUTHENTICATION_BACKENDS = ('smartmin.backends.CaseInsensitiveBackend',)
+AUTHENTICATION_BACKENDS = (
+    'smartmin.backends.CaseInsensitiveBackend',
+    'guardian.backends.ObjectPermissionBackend',
```

Comment:
`./manage.py check` tests were failing on Travis without this. Perhaps a dependency changed to make this required?